README.md

```diff
@@ -194,9 +194,9 @@ Emissions to hot flows that don't have active consumers are dropped. It's import
 (and therefore have an active collector) on a flow _before_ emissions to a flow are made. For example:
 
 ```kotlin
-val mutableStateFlow = MutableStateFlow<Int>(replay = 0)
-mutableStateFlow.emit(1)
-mutableStateFlow.test {
+val mutableSharedFlow = MutableSharedFlow<Int>(replay = 0)
```

Comment:
Despite my original comments, can we change this to state flow? You can correct the param if you'd like. I imagine people use state flow more frequently than shared flow, and thus would like that to be the explicit example.

Reply:
Whoops too late. New PR?

Reply:
@JakeWharton womp! Saw you merged. This isn't a big deal.

Reply:
Yeah I can fix it up. One second. Pr incoming.